src/Aspire.Hosting.Azure.Storage/AzureBlobStorageResource.cs

```diff
@@ -42,10 +63,29 @@ void IResourceWithAzureFunctionsConfig.ApplyAzureFunctionsConfiguration(IDiction
             // uses the queue service for its internal bookkeeping on blob triggers.
             target[$"{connectionName}__blobServiceUri"] = Parent.BlobEndpoint;
             target[$"{connectionName}__queueServiceUri"] = Parent.QueueEndpoint;
+
             // Injected to support Aspire client integration for Azure Storage.
             // We don't inject the queue resource here since we on;y want it to
             // be accessible by the Functions host.
             target[$"{AzureStorageResource.BlobsConnectionKeyPrefix}__{connectionName}__ServiceUri"] = Parent.BlobEndpoint;
+
+            if (blobContainerName is not null)
+            {
+                target[$"{AzureStorageResource.BlobsConnectionKeyPrefix}__{connectionName}__BlobContainerName"] = blobContainerName;
+            }
         }
```

Comment:
Is this a thing?

Reply:
I have no idea, tbh... I'm not sure how to test this with AF. @captainsafia any hints?

Reply:
Chatted with @captainsafia, it's likely not a thing. I will test AF-integrations once the client integrations are completed.